### ISSUE TITLE
Livesync with --bundle flag often dies after .scss changes

### ISSUE THREAD

**Issue by NathanWalker** [NativeScript/nativescript-dev-webpack#563]
## Issue Checklist

- [x] You are submitting: a bug report
- [x] You verified that this is a real problem by searching the [NativeScript Forum] and the other [issues] in this repo.
- [x] You checked our [demo apps] and the [documentation] for sample usage.

## Tell us about the problem

Using the `--bundle` flag with livesync will often die after a few changes. Sometimes only after 1 change. For some reason .scss changes seem to be the most susceptible.

## Local environment

- [x] Platform Mac os High Sierra 10.3.5
- [x] Node version 10.3.0, npm 5.6 (and confirmed on 6.1 as well)
- [x] NativeScript CLI version 4.1 {N} installed globally and using 4.1 runtimes.

## Project data

- [x] Platform you are building the app for: Both
- [x] Node dependencies:

I will provide the dev dependencies since a sample project was emailed to @sis0k0 + @vakrilov 👍 

```
"@angular-devkit/core": "~0.6.3",
    "@angular/compiler-cli": "~6.0.0",
    "@angular/language-service": "~6.0.0",
    "@ngtools/webpack": "~6.0.3",
    "angular-router-loader": "~0.8.1",
    "babel-traverse": "6.26.0",
    "babel-types": "6.26.0",
    "babylon": "6.18.0",
    "clean-webpack-plugin": "~0.1.19",
    "codelyzer": "~4.2.1",
    "copy-webpack-plugin": "~4.5.1",
    "css-loader": "~0.28.11",
    "extract-text-webpack-plugin": "~3.0.2",
    "lazy": "1.0.11",
    "nativescript-css-loader": "~0.26.0",
    "nativescript-dev-typescript": "~0.7.0",
    "nativescript-dev-webpack": "~0.12.0",
    "nativescript-worker-loader": "~0.9.0",
    "raw-loader": "~0.5.1",
    "resolve-url-loader": "~2.3.0",
    "sass-loader": "^7.0.2",
    "tns-platform-declarations": "~4.1.0",
    "typescript": "2.7.2",
    "uglifyjs-webpack-plugin": "~1.2.5",
    "webpack": "~4.6.0",
    "webpack-bundle-analyzer": "~2.13.0",
    "webpack-cli": "~2.1.3",
    "webpack-parallel-uglify-plugin": "~1.1.0",
    "webpack-sources": "~1.1.0"
```

- [c] Webpack configuration:

latest from nativescript-dev-webpack

iOS will die and leave no stack.
However Android dies with this stack:

```
An uncaught Exception occurred on "main" thread.
java.lang.RuntimeException: Unable to start activity ComponentInfo{io.knowledge.triviaspardev/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: 
Calling js method onCreate failed

TypeError: Cannot read property 'create' of undefined
File: "file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js, line: 11035, column: 14

StackTrace: 
Frame: function:'', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 11035, column: 15
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/frame-common.js.FrameBase.navigate', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 18274, column: 30
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/frame.js.ActivityCallbacksImplementation.setActivityContent', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 19510, column: 30
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/frame.js.ActivityCallbacksImplementation.onCreate', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 19369, column: 14
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/activity.js.NativeScriptActivity.onCreate', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 17370, column: 25


at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2927)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2988)
at android.app.ActivityThread.-wrap14(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1631)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:154)
at android.app.ActivityThread.main(ActivityThread.java:6682)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1520)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1410)
Caused by: com.tns.NativeScriptException: 
Calling js method onCreate failed

TypeError: Cannot read property 'create' of undefined
File: "file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js, line: 11035, column: 14

StackTrace: 
Frame: function:'', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 11035, column: 15
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/frame-common.js.FrameBase.navigate', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 18274, column: 30
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/frame.js.ActivityCallbacksImplementation.setActivityContent', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 19510, column: 30
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/frame.js.ActivityCallbacksImplementation.onCreate', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 19369, column: 14
Frame: function:'push.../../../node_modules/tns-core-modules/ui/frame/activity.js.NativeScriptActivity.onCreate', file:'file:///data/data/io.knowledge.triviaspardev/files/app/vendor.js', line: 17370, column: 25


at com.tns.Runtime.callJSMethodNative(Native Method)
at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1100)
at com.tns.Runtime.callJSMethodImpl(Runtime.java:982)
at com.tns.Runtime.callJSMethod(Runtime.java:969)
at com.tns.Runtime.callJSMethod(Runtime.java:953)
at com.tns.Runtime.callJSMethod(Runtime.java:945)
at com.tns.NativeScriptActivity.onCreate(NativeScriptActivity.java:18)
at android.app.Activity.performCreate(Activity.java:6942)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1126)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2880)
... 9 more
```

/cc @davecoffin @shiv19

**Comment by mgambati** [comment]
I'm with the exact same error. Windows 10 and Android.

**Comment by IAMtheIAM** [comment]
For me livesync with webpack does not work at all. Webpack detects the changes, rebuilds, all the hooks execute for tns, and then it installs the app... but it installs the same app (old code). I have not figured out whats going on

`tns run android --bundle` was all we need to do right?

**Comment by sis0k0** [comment]
Hi, @IAMtheIAM, this is an old issue that must have been resolved in v0.14.0. Can you open another one and provide more info about your local setup there?
